packages/now-cli/test/integration.js

```diff
@@ -126,6 +126,36 @@ if (!process.env.CI) {
   );
 }
 
+function mockLoginApi(req, res) {
+  const { url = '/', method } = req;
+  let { pathname = '/', query = {} } = parseUrl(url, true);
+  console.log(`[mock-login-server] ${method} ${pathname}`);
+  const securityCode = 'Bears Beets Battlestar Galactica';
+  if (
+    method === 'POST' &&
+    pathname === '/now/registration' &&
+    query.mode === 'login'
+  ) {
+    res.end(JSON.stringify({ token, securityCode }));
+  } else if (
+    method === 'GET' &&
+    pathname === '/now/registration/verify' &&
+    query.email === email
+  ) {
+    res.end(JSON.stringify({ token }));
+  } else {
+    res.statusCode = 405;
+    res.end(JSON.stringify({ code: 'method_not_allowed' }));
+  }
+}
+
+const LOGIN_API_PORT = 2999;
```

Comment:
Could we instead pass `0` to` `listen()` to have an ephemeral port assigned from the OS?

Reply:
Done 👍